docs/reference/rockcraft.yaml.rst

```diff
@@ -147,6 +147,23 @@ specification syntax exactly, with each entry defining a Pebble service. For
 each service, the ``override`` and ``command`` fields are mandatory, but all
 others are optional.
 
+``entrypoint-service``
+------------------------
+
+**Type**: string
+
+**Required**: No
+
+Makes Rockcraft extend the default OCI entrypoint
```

Comment:
I think this description is missing a link between the field and the pebble service. Something like `The optional name of the Pebble service to serve as the OCI entrypoint. If set, this makes Rockcraft extend...`